test/contracts/AuctionRegistrar.cpp

```diff
@@ -425,63 +429,32 @@ BOOST_AUTO_TEST_CASE(auction_bidding)
 	string name = "x";
 
 	unsigned startTime = 0x776347e2;
-	m_rpc.test_modifyTimestamp(startTime);
+	m_evmHost->m_state.timestamp = startTime;
 
 	RegistrarInterface registrar(*this);
 	// initiate auction
 	registrar.setNextValue(8);
 	registrar.reserve(name);
 	BOOST_CHECK_EQUAL(registrar.owner(name), 0);
 	// overbid self
-	m_rpc.test_modifyTimestamp(startTime + m_biddingTime - 10);
+	m_evmHost->m_state.timestamp = startTime + m_biddingTime - 10;
 	registrar.setNextValue(12);
 	registrar.reserve(name);
 	// another bid by someone else
 	sendEther(account(1), 10 * ether);
 	m_sender = account(1);
-	m_rpc.test_modifyTimestamp(startTime + 2 * m_biddingTime - 50);
+	m_evmHost->m_state.timestamp = startTime + 2 * m_biddingTime - 50;
 	registrar.setNextValue(13);
 	registrar.reserve(name);
 	BOOST_CHECK_EQUAL(registrar.owner(name), 0);
 	// end auction by first bidder (which is not highest) trying to overbid again (too late)
 	m_sender = account(0);
-	m_rpc.test_modifyTimestamp(startTime + 4 * m_biddingTime);
+	m_evmHost->m_state.timestamp = startTime + 4 * m_biddingTime;
 	registrar.setNextValue(20);
 	registrar.reserve(name);
 	BOOST_CHECK_EQUAL(registrar.owner(name), account(1));
 }
 
-BOOST_AUTO_TEST_CASE(auction_renewal)
```

Comment:
This test is just removed?

Reply:
Because I could not make it work and it is extremely outdated anyway. The problem was that time now progresses a little differently and we do not have the distinction between pending and sealed blocks.